library/src/main/java/com/yuyh/library/EasyGuide.java

```diff
@@ -1,384 +1,416 @@
+/*
```

Comment:
Applibgroup copyright should not be added for any of the lib files unless the files which are implemented from scratch. Please check and correct this for all lib files.